.github/workflows/run.yml

```diff
@@ -4,12 +4,27 @@ on:
   schedule:
     - cron: '2 7 * * *'
 jobs:
-  run:
+
```

Comment:
Unnecessary line break

Reply:
Three days away and I forget the line breaks :smile: I've removed it.